docs/languages/ruby.md

```diff
@@ -16,7 +16,7 @@ This topic goes into detail about setting up and using Ruby within Visual Studio
 
 ### Install Ruby through a version manager
 
-While Ruby is installed by default on some operating systems (such as macOS and some distributions of Linux), we recommend using a version manager such as [rbenv](https://github.com/rbenv/rbenv) to be able to access newer versions of Ruby on Windows, macOS, and Linux. Follow the [installation guidance](https://github.com/rbenv/rbenv#installation) for your platform.
+While Ruby is installed by default on some operating systems (such as macOS and some distributions of Linux), we recommend using a version manager such as [rbenv](https://github.com/rbenv/rbenv) to be able to access newer versions of Ruby on macOS and Linux. Follow the [installation guidance](https://github.com/rbenv/rbenv#installation) for your platform. Because [rbenv] doesn't work for Windows, it is recommended to install via [RubyInstallers](https://rubyinstaller.org/downloads/) directly.
```

Comment:
As mentioned on the Ruby language site, for Windows there's `rbenv for windows` (https://www.ruby-lang.org/en/documentation/installation/#rbenv-for-windows). This provides a consistent cmdline experience across OSes, allowing easily switching between versions.

Can you update accordingly?